cmd/lansible

```diff
@@ -314,9 +314,16 @@ EOF
   elif [[ "$@" == *--playbook* ]] || [[ "$@" == *--with-playbook* ]]; then
     strap::bot "Run Ansible Playbook"
     strap::ansible::playbook::run "$@"
-  elif [[ -d "${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default" ]]; then
-    strap::bot "Run Ansible Playbook"
-    strap::ansible::playbook::run --with-playbook="${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default" "$@" 
+  elif [[ -z "$@" ]]; then
+    if [[ -f "${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml" ]]; then
+      strap::bot "Run Ansible Playbook"
+      strap::ansible::playbook::run --with-playbook="${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml" "$@"
+    else
+      printf "\nDefault playbook ${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml not found. No ansible playbook will be run."
+      printf "\nPlease rerun the command under a directory that has .strap/ansible/playbooks/default/main.yml.\n"
+    fi
```

Comment:
Do we need to exit with a non-zero exit code here?

Reply:
I have thought about it and I think we can still exit 0 since this basically means no default playboook here, only core strap will be run.

Reply:
These messages look more like an error though - we shouldn't ask them to re-run anything if it's a successful exit/use case, no?

Reply:
It is still a use case when people just ran `strap lansible` in somewhere like `~` where there isn't a default playbook. Though your mentioning of the error message is a good point. It's better to change it to.
```
printf "\nDefault playbook ${STRAP_WORKING_DIR}/.strap/ansible/playbooks/default/main.yml not found. No ansible playbook will be run."
      printf "\nIf you want to run a default playbook, run `strap lansible` under a directory that has .strap/ansible/playbooks/default/main.yml\n"
```